process_pr.py

```diff
@@ -109,7 +109,7 @@ def format(s, **kwds):
 CODE_CHECKS_REGEXP = re.compile(
     r"code-checks(\s+with\s+cms.week[0-9].PR_[0-9a-f]{8}/[^\s]+|)(\s+and\s+apply\s+patch|)$"
 )
-WF_PATTERN = r"[1-9][0-9]*(\.[0-9]+|)"
+WF_PATTERN = r"[a-z1-9][a-z0-9_]*(\.[0-9]+|)"
```

Comment:
How about `(?:[a-z][a-z0-9_]*"[1-9][0-9]*(\.[0-9]+)|[1-9][0-9]*(\.[0-9]+|))`? Or do we want to allow list names to contain `.`?

Reply:
`(?:...)` is a non-capturing group to avoid shifting group numbers

Reply:
yes I did not want to change the number of matching groups. Your suggestion sounds good. I will update it along with other changes we might need in the pr test part itself